client/blocks/comments/comment-count.jsx

```diff
@@ -28,7 +27,11 @@ const CommentCount = ( { count, translate } ) => {
 	return (
 		<div className="comments__comment-count">
 			<span className="comments__comment-count-phrase">{ countPhrase }</span>
-			{ count === 0 && translate( ' - Add the first! ' ) }
+			{ count === 0 &&
```

Comment:
Why are we actually string concatenating here and not further up?
i.e. change `countPhrase = translate( 'No comments' );` to `countPhrase = translate( 'No comments - add the first!' );`

Reply:
So we can apply a style to just the 'no comments' part only (which currently applies `text-transform: uppercase` in Reader full post).